docs/frontity-cli/serve.md

```diff
@@ -0,0 +1,25 @@
+# `serve`
+
+Starts a server in production mode.
+
+```shell
+npx frontity serve [options]
+```
+
+## Arguments
+
+### **`[options]`**
+
+|            Option            | Description                                                                                                                                  |
+| :--------------------------: | -------------------------------------------------------------------------------------------------------------------------------------------- |
+|        `--port <port>`         | Runs the server on a custom port. Default is 3000.                                                                                      |
+|         `-s`, `--https`          | Runs the server using https. |
+| `-h`, `--help`  | output usage information                                                                                                             |
+
+## Examples
+
+- Starts a server (using the code in the `build` folder generated by the `npx frontity build` command) using https and port 3002
```

Comment:
Let's put this info in the main description above as well (that you have to build the code with `npx frontity build`) first, otherwise this command won't work! :)